synapse/models/biz.py

```diff
@@ -102,9 +102,27 @@ def getModelDefs(self):
                     ('buyer', ('ps:contact', {}), {
                         'doc': 'The primary contact information for the buyer.',
                     }),
+                    ('buyer:org', ('ou:org', {}), {
+                        'doc': 'The buyer org.',
+                    }),
+                    ('buyer:orgname', ('ou:name', {}), {
```

Comment:
I probably just missed the modeling conversation, but this seems bit redundant with `ou:org:name` and `ou:org:url` (same for the corresponding seller/madeby ones)?

Reply:
It's for entity resolution where you may have observables that contain an org name or fqdn but you dont have a corresponding org yet. recording them so they can be glued back up later.

Reply:
:+1: